README.adoc

```diff
@@ -0,0 +1,203 @@
+= Secure Xamarin Template
+
+image:https://img.shields.io/badge/Android-Compatible-green.svg[]
+image:https://img.shields.io/badge/iOS-Compatible-green.svg[]
+
+[width="85%"]
+|===========================================================================================================
+|                 | *Project Info*
+| License:        | Apache License, Version 2.0
+| Build:          | msbuild
+| End User Documentation:  | link:https://docs.aerogear.org[]
+| Community Documentation: | link:https://aerogear.org[]
+| Issue tracker:  | link:https://issues.jboss.org/browse/AEROGEAR[]
+| Mailing lists:  | link:https://groups.google.com/forum/#!forum/aerogear[aerogear-dev]
+|===========================================================================================================
+
+A Xamarin Application that demonstrates features of the Aerogear Xamarin SDK on iOS and Android.
+
+For more information, please check the https://github.com/feedhenry/mobile-security[Mobile Security Project].
+
+== Features
+- OpenID Connect Authentication
+- Single Sign-on
+- Device Trust Checks (Jailbreak/ROOT, etc)
+- 2FA with OTP (via Keycloak)
+- Authentication Brute Force Detection (via Keycloak)
+- Account Lockout Policies (via Keycloak)
+- Authentication/Access Control Auditing & Logging (via Keycloak)
+- Authenticated Calls to Protected Endpoints (via Keycloak)
```

Comment:
I think we can reduce the features here to match the services we have in AeroGear. So I think perhaps we can use:

* AeroGear Identity Management Service
* Single Sign On
* Device Security Checks

and those should link to the appropriate section in https://docs.aerogear.org/